cmd/nodecmd/create.go

```diff
@@ -443,6 +449,20 @@ func requestAWSAccountAuth() error {
 	return nil
 }
 
+func requestStopAWSNodeAuth() error {
+	ux.Logger.PrintToUser("Do you authorize Avalanche-CLI to access your AWS account to stop your Avalanche Validator node?")
```

Comment:
is this stopping or deleting?

We should say that this could cause Subnet instability and is not reversible?